### DISCUSSION TITLE
Add invert color for B&W mode

### DISCUSSION THREAD

**Discussion by mykeehu** [visioncortex/vtracer#72] [Ideas]
I have a greyscale image, and if I want to process it as black and white instead of colour, the background will be black and the lines will be white. It would be nice to have a switch that inverts the line colors, so instead of white lines, it saves black to SVG.

**Comment by tyt2y3** [comment]
I think you can use ffmpeg to flip the image, it's quite neat. https://superuser.com/questions/1336315/invert-colors-of-a-video-using-ffmpeg
